frontend/frontend-auto-translate.php

```diff
@@ -141,7 +141,7 @@ public function translate_included_ids_in_query( $query ) {
 		// According to the codex, this type of query is deprecated as of WP 3.1 but it does not appear in WP 3.5 source code
 		foreach ( array_intersect( $this->model->get_translated_taxonomies(), get_taxonomies( array( '_builtin' => false ) ) ) as $taxonomy ) {
 			$tax = get_taxonomy( $taxonomy );
-			if ( ! empty( $tax ) && ! empty( $qv[ $tax->query_var ] ) ) {
+			if ( ! empty( $tax ) && ! empty( $tax->query_var ) && ! empty( $qv[ $tax->query_var ] ) ) {
```

Comment:
Instead of checking the key, shouldn't we check the value's type?

```suggestion
			if ( ! empty( $tax ) && ! empty( $qv[ $tax->query_var ] ) && is_string( $qv[ $tax->query_var ] ) ) {
```

Reply:
Some value type check is already made in `translate_terms_list()`: https://github.com/polylang/polylang/blob/9a22f97915b4f99272220e3d849c43c30cdfad9a/frontend/frontend-auto-translate.php#L314-L319
It can be an array or a string. It would make sense to improve it there.

I still believe that we need to check the key anyway, not to mix a taxonomy without query var with some extra data added by any 3rd party. There would still be a risk that this extra data could  pass the check is if it's a string or an array.